tap_linkedin_ads/client.py

```diff
@@ -114,7 +114,11 @@ def raise_for_error(response):
     message = "HTTP-error-code: {}, Error: {}".format(
                 error_code, error_description)
 
-    exc = ERROR_CODE_EXCEPTION_MAPPING.get(error_code, {}).get("raise_exception", LinkedInError)
+    if error_code not in ERROR_CODE_EXCEPTION_MAPPING and error_code > 500:
```

Comment:
I know a 500 error is in the Error code dict, but could `error_code > 500` be changed to `error_code >= 500` just so we're not dependent on it being in the dict.

Reply:
Got it @bryantgray . Replaced `error_code > 500` with `error_code >= 500`.